apps/src/code-studio/components/progress/ScriptOverviewTopRow.jsx

```diff
@@ -102,6 +105,7 @@ class ScriptOverviewTopRow extends React.Component {
               size={Button.ButtonSize.large}
               style={{marginLeft: 10}}
             />
+            {assignedSectionId && <Assigned />}
```

Comment:
Should this be behind the experiment?

Also maybe I'm missing something but I'm not sure how this isn't going to show on any script that a student is on?

Reply:
Yes, updated so this is behind the experiment. 

This will show up for students on the script overview _if_ that script is assigned to them (by any teacher for any of the sections they are in) because of this line: https://github.com/code-dot-org/code-dot-org/blob/62aafa978ad0be3b4ae441604874b21cb70fc204/dashboard/app/models/script.rb#L1336 which was in an earlier PR. If that script isn't assigned to the current student, there won't be an assignedSectionId.

Reply:
👍